app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results_value_cashe = {}
+
+    def wrapper(*args, **kwargs) -> Callable:
```

Comment:
Good to add an annotation, that this is a wrapped function.
```suggestion
    @wraps(func)
    def wrapper(*args) -> Callable:
```

Reply:
changes were created